src/pages/beta/[viewType]/key-stages/[keyStageSlug]/subjects/[subjectSlug]/programmes.tsx

```diff
@@ -16,27 +14,22 @@ import {
 } from "../../../../../../../node-lib/isr";
 import { VIEW_TYPES, ViewType } from "../../../../../../../common-lib/urls";
 import getPageProps from "../../../../../../../node-lib/getPageProps";
+import curriculumApi2023 from "../../../../../../../node-lib/curriculum-api-2023";
+import { ProgrammeListingPageData } from "../../../../../../../node-lib/curriculum-api-2023/queries/programmeListing/programmeListing.schema";
 
-export type ProgrammeListingPageProps = TierListingData;
+const ProgrammesListingPage: NextPage<ProgrammeListingPageData> = (props) => {
+  const {
+    programmes,
+    keyStageSlug,
+    subjectSlug,
+    keyStageTitle = "Key stage 3",
+  } = props;
+  const subjectTitle = programmes[0]?.subjectTitle ?? "";
```

Comment:
I'm going to add this as a field in the row on the MV so we destructure it with the other props. https://www.notion.so/oaknationalacademy/I-want-to-connect-the-programme-listing-MV-to-the-2023-curriculum-API-ebc22f117c434ef4bb917ece325fe1c7